<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPathBundles</class>
 <widget class="QWidget" name="SettingsPathBundles">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Folder to search Bundles</string>
  </property>
  <widget class="QPushButton" name="btn_add">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="pathBundleEntries">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>469</width>
     <height>199</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="gridSize">
    <size>
     <width>0</width>
     <height>22</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
